pkg/api/server/v1alpha2/records_test.go

```diff
@@ -602,3 +603,53 @@ func TestDeleteRecord(t *testing.T) {
 		}
 	})
 }
+
+// TestListRecords_multiresult tests listing records across multiple result parents.
```

Comment:
Since we don't support List cross parents, "across multiple result parents" might be a kind of misleading?

Reply:
Reworded. We're listing across multiple record parents (i.e. results), but not over multiple result parents.